src/optimize.ts

```diff
@@ -170,6 +175,19 @@ async function analyse(file: string): Promise<void> {
   }
 }
 
+function isImgAboveTheFold($: cheerio.CheerioAPI, imgElement: cheerio.Element, theFold: number) {
+  const aboveTheFoldAttr: string | number | undefined = $(imgElement).attr('above-the-fold');
```

Comment:
Maybe I would use `data-abovethefold` just because `data-` attributes are more "standard" for these things.

Reply:
sure